A/AbstractPlotting/Compat.toml

```diff
@@ -10,25 +14,42 @@ SignedDistanceFields = "0"
 StaticArrays = "0"
 UnicodeFun = "0"
 
-["0-0.9.0"]
-GeometryTypes = "0.4.5-0"
-Reactive = "0"
-
 ["0.0"]
 Fontconfig = "0"
 julia = "0.6-0"
 
-["0.9-0"]
+["0.9-0.9.7"]
 PlotUtils = "0"
 Showoff = "0"
 julia = "0.7-1"
 
-["0.9.1-0"]
+["0.9.1-0.9.7"]
 Contour = "0"
 FileIO = "0-1"
 FixedPointNumbers = "0-0.5"
 GeometryTypes = "0.7.2-0"
 Observables = "0"
 
-["0.9.2-0"]
+["0.9.2-0.9.7"]
 ImageMagick = "0"
+
+["0.9.8-0"]
+ColorBrewer = "*"
```

Comment:
Why include deps like this in your `[compat]` at all?

Reply:
To remove the overly strict bounds that General put on them, which caused lots of problems. Or did I misinterpret how to fix that? I thought I can only overwrite it with a new compat entry, and since those packages basically are compatible from their first version on until eternity, that's the compat I gave them ;)

Reply:
yes that was the idea... Maybe I misunderstood, that I need to give a package a compat entry, to overwrite the overly strict ones from the registry?

Reply:
I mean, leaving it out completely from the `[compat]` list is equivalent to what you have here, but what you have here is just a lot uglier :)

What do you mean with overly strict ones? Inherited from METADATA?

Reply:
I'm not sure how they got in there, I just know that I suddenly had lots of bounds problems with my packages, even though I don't even bound most of my deps. And I thought I read on slack, that one has to put down new bounds for dependencies to fix this - Since I don't actually have & need any bounds, I thought that's what I need to do :D

Reply:
Sure, I just think it would be better if you completely excluded them from your `[compat]` list.

Reply:
Well, that doesn't sound to me like "giving them a new bound" :D but if it has the same effect, of course, that's what I should do!

Reply:
But you are, since no bounds is a new bound.

Reply:
Well, it's not like the registry was respecting that before... Anyways, if you say so, I'll update this, since all those `">= 0"` entries are pretty ugly.